ansys/dpf/core/elements.py

```diff
@@ -511,8 +511,8 @@ def materials_field(self):
         >>> from ansys.dpf.core import examples
         >>> model = dpf.Model(examples.static_rst)
         >>> elements = model.metadata.meshed_region.elements
-        >>> elements.materials_field.data
-        array([1, 1, 1, 1, 1, 1, 1, 1])
+        >>> print(elements.materials_field.data)
+        [1 1 1 1 1 1 1 1]
```

Comment:
It's preferable in these examples to use the `__repr__` rather than `print`.

Recommend revert.